docs/api/zod/coerceFormValue.md

```diff
@@ -0,0 +1,152 @@
+# unstable_coerceFormValue
+
+A helper that enhances the schema with extra preprocessing steps to strip empty value and coerce form value to the expected type.
+
+```ts
+const enhancedSchema = coerceFormValue(schema, options);
+```
+
+The following rules will be applied by default:
+
+1. If the value is an empty string / file, pass `undefined` to the schema
+2. If the schema is `z.number()`, trim the value and cast it with the `Number` constructor
+3. If the schema is `z.boolean()`, treat the value as `true` if it equals to `on` (Browser default `value` of a checkbox / radio button)
+4. If the schema is `z.date()`, cast the value with the `Date` constructor
+5. If the schema is `z.bigint()`, cast the value with the `BigInt` constructor
```

Comment:
`trim` is used in the same way as `Number`, so would this be a good explanation?

```suggestion
5. If the schema is `z.bigint()`, trim the value and cast it with the `BigInt` constructor
```